.github/workflows/linting.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           restore-keys: ${{ runner.os }}-linting-${{ hashFiles('**/poetry.lock') }}
 
       - run: |
-          pip install bandit black codespell mypy==0.982 safety pylint==2.15.2 packaging==22 ruff==0.0.243
+          pip install bandit black codespell mypy==0.982 safety pylint==2.15.2 packaging==22 ruff==0.0.246
```

Comment:
can you remove safety and packaging from this line.  They aren't needed and I keep forgetting